guides/doc-Administering_Project/master.adoc

```diff
@@ -66,6 +66,9 @@ include::common/assembly_using-foreman-webhooks.adoc[leveloffset=+1]
 
 include::common/assembly_searching-and-bookmarking.adoc[leveloffset=+1]
 
+[appendix]
+include::common/modules/proc_troubleshooting-permission-issues.adoc[leveloffset=+1]
+
```

Comment:
"Managing users and roles" seems like a better fit because that assembly covers procedure-related permissions.